typed-racket-lib/typed-racket/typecheck/tc-toplevel.rkt

```diff
@@ -6,15 +6,15 @@
          (prefix-in c: (contract-req))
          (rep type-rep)
          (types utils abbrev type-table struct-table)
-         (private parse-type type-annotation syntax-properties)
+         (private parse-type type-annotation syntax-properties type-contract)
          (env global-env init-envs type-name-env type-alias-env
               lexical-env env-req mvar-env scoped-tvar-env
               type-alias-helper)
          (utils tc-utils)
-         (typecheck provide-handling def-binding tc-structs
-                    typechecker internal-forms)
+         "provide-handling.rkt" "def-binding.rkt" "tc-structs.rkt"
+         "typechecker.rkt" "internal-forms.rkt"
```

Comment:
Is there any reason to make these relative paths here? (and if so, maybe a separate commit?)